### PR TITLE
feat(react-native): rn 0.72.7 update

### PR DESCRIPTION
**Pull request by Calinteodor**

### CODE REVIEW

File: android/settings.gradle [hunk 1]
```diff
@@ -1,10 +1,10 @@
 rootProject.name = 'jitsi-meet'
 
 include ':app', ':sdk'
-includeBuild('../node_modules/react-native-gradle-plugin')
+includeBuild('../node_modules/@react-native/gradle-plugin')
```
Comment (saghul) [inline android/settings.gradle:4]: Come to think of it, we might not need this at all.

**Review by saghul — DISMISSED** [review]
Left some comments.

File: android/app/build.gradle [hunk 2]
```diff
@@ -75,7 +75,6 @@ android {
 }
 
 dependencies {
-    implementation fileTree(dir: 'libs', include: ['*.jar'])
```
Comment (saghul) [inline android/app/build.gradle:78]: Don't we need this at all?

Comment (saghul) [reply android/app/build.gradle:78]: Ack, away it goes!

File: ios/Podfile [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
+require Pod::Executable.execute_command('node', ['-p',
+  'require.resolve(
+    "react-native/scripts/react_native_pods.rb",
+    {paths: [process.argv[1]]},
+  )', __dir__]).strip
 
 platform :ios, '12.4'
 workspace 'jitsi-meet'
 
-install! 'cocoapods', :deterministic_uuids => false
```
Comment (saghul) [inline ios/Podfile:7]: Why is this line no longer valid?

File: ios/.xcode.env [hunk 4]
```diff
@@ -0,0 +1 @@
+export NODE_BINARY=$(command -v node)
```
Comment (saghul) [inline ios/.xcode.env:1]: Is this used anywhere?

File: ios/Podfile (hunk 3 above)
Comment (Calinteodor) [reply ios/Podfile:7]: This was replaced.

File: ios/.xcode.env (hunk 4 above)
Comment (Calinteodor) [reply ios/.xcode.env:1]: I am not sure, this was automatically generated, but I tend to not think so. Will check it.

File: android/sdk/build.gradle [hunk 5]
```diff
@@ -49,7 +49,8 @@ dependencies {
     implementation 'com.jakewharton.timber:timber:4.7.1'
     implementation 'com.squareup.duktape:duktape-android:1.3.0'
     implementation 'com.google.code.gson:gson:2.8.6'
-    implementation "androidx.startup:startup-runtime:1.1.0"
+    implementation 'androidx.startup:startup-runtime:1.1.0'
+    implementation 'com.facebook.react:react-android:0.72.7'
```
[the remark this answers is not in the document]
Comment (saghul) [reply android/sdk/build.gradle:53]: Or better, pleace this is its stead.

File: ios/.xcode.env (hunk 4 above)
Comment (keremoge) [reply ios/.xcode.env]: [react-native-xcode.sh](https://github.com/facebook/react-native/blob/55e4543eea1a2f7a42ed6997c9a692b4fc61550a/packages/react-native/scripts/react-native-xcode.sh#L142) does use it, 
--keep in mind that I had problems with using multiple node env (nvm)-- maybe about my environment

**Review by saghul — APPROVED** [review]

**Pull request merged**